fed/barriers.py

```diff
@@ -260,7 +260,7 @@ async def is_ready(self):
         return True
 
     async def get_data(self, upstream_seq_id, curr_seq_id):
-        logger.debug(
```

Comment:
Shall we keep `debug`?  Since `info` is noisy to user.

Reply:
Yes. It was introduced unintended.